src/camera.py

```diff
@@ -0,0 +1,76 @@
+from os import getcwd
+
+from picar.SunFounder_PCA9685 import Servo
+from picar import filedb
+
+
+'''
+Camera class.
+
+Uses SunFounder Servo class to control camera servos allowing for pan and tilt.
+Uses SunFounder filedb to configure offset for pan and tilt functions.
+'''
+class Camera:
+    PAN_CHANNEL = 1
+    TILT_CHANNEL = 2
+
+    def __init__(self, bus_number=1):
+        self._DB = filedb.fileDB(getcwd() + "/../SunFounder_PiCar-V/remote_control/remote_control/driver/config")
+        self._pan_offset = int(self._DB.get('pan_offset', default_value=0))
+        self._tilt_offset = int(self._DB.get('tilt_offset', default_value=0))
+        self._pan_servo = Servo.Servo(self.PAN_CHANNEL, bus_number=bus_number, offset=_self.pan_offset)
+        self._tilt_servo = Servo.Servo(self.TILT_CHANNEL, bus_number=bus_number, offset=_self.tilt_offset)
+
+
+    '''
+    Controls the horizontal movement of the camera.Usable inputs are [0,180]. 0 refers to the 
+    vehicles righthand side while 180 refers to the vehicles lefthand side. 
+    '''
+    def pan(self, degree):
+        self._pan_servo.write(self.validate_degree(degree,180))
+
+
+    '''
```

Comment:
This isn't true. It doesn't write to a config within the current directory. Also this isn't true. Adding an offset of 10 and an offset of 100 should both move the camera in the same direction because you're just adding the offset. This is what the SunFounder API does, which I told you not to copy. Our system should be absolute, not relative. Fix the comments to be true, fix the line lengths to fit 80 characters, and fix the function.

As it is, I can pass in 100000 and it'll take it.